ext/GDBM_File/GDBM_File.xs

```diff
@@ -145,15 +145,15 @@ output_datum(pTHX_ SV *arg, char *str, int size)
 #define gdbm_setopt(db,optflag,optval,optlen) not_here("gdbm_setopt")
 #endif
 
-#ifndef GDBM_ITEM_NOT_FOUND
-# define GDBM_ITEM_NOT_FOUND GDBM_NO_ERROR
+#if GDBM_VERSION_MAJOR == 1 && GDBM_VERSION_MINOR < 13
+/* Prior to 1.13, only gdbm_fetch set GDBM_ITEM_NOT_FOUND if the requested
+   key did not exist.  Other similar function wouls set GDBM_NO_ERROR instead.
```

Comment:
Typos: Should say "...functions would..."

Reply:
Done in next commit to branch; also next typo.